### DISCUSSION TITLE
Support for Additional HTTP Methods in zap.Endpoint.Listener

### DISCUSSION THREAD

**Discussion by ghost** [zigzap/zap#133] [Q&A]
Hi,

Is there a way to add support for additional (webdav specific) HTTP methods like PROPFIND, COPY, MOVE, LOCK, and UNLOCK...  when using zap.Endpoint.Listener? I haven't been able to figure out how to handle these methods, and any guidance would be appreciated.

Thanks in advance for your help!

**Comment by renerocksai** [comment]
Hi,

You would need to add support for them in zap's endpoint.zig. It's just
a few lines of boilerplate; easy to recognize the pattern.
Best, Rene

On September 26, 2024, Rene Schallner ***@***.***> wrote:
> Hi,
>
> Is there a way to add support for additional (webdav specific) HTTP
> methods like PROPFIND, COPY, MOVE, LOCK, and UNLOCK... when using
> zap.Endpoint.Listener? I haven't been able to figure out how to handle
> these methods, and any guidance would be appreciated.
>
> Thanks in advance for your help!
>

—
Reply to this email directly, view it on GitHub
<https://github.com/orgs/zigzap/discussions/133>, or unsubscribe
<https://github.com/notifications/unsubscribe-
auth/AHLWBJ7GTW4TUA4GXV5J5JDZYRXTRAVCNFSM6AAAAABO52K2CKVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZXGI2DCMBTG4>.
You are receiving this because you are subscribed to this thread.Message
ID: ***@***.***>
